pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -3452,6 +3452,131 @@ func TestIsRunning(t *testing.T) {
 			CustomTask:   true,
 		},
 		want: false,
+	}, {
+		name: "matrixed taskruns not started",
+		rprt: ResolvedPipelineRunTask{
+			PipelineTask: matrixedPipelineTask,
+		},
+		want: false,
+	}, {
+		name: "matrixed taskruns running",
+		rprt: ResolvedPipelineRunTask{
+			PipelineTask: matrixedPipelineTask,
+			TaskRuns:     []*v1beta1.TaskRun{makeStarted(trs[0]), makeStarted(trs[1])},
+		},
+		want: true,
+	}, {
+		name: "one matrixed taskrun running",
+		rprt: ResolvedPipelineRunTask{
+			PipelineTask: matrixedPipelineTask,
+			TaskRuns:     []*v1beta1.TaskRun{makeStarted(trs[0]), makeSucceeded(trs[1])},
+		},
+		want: true,
+	}, {
+		name: "matrixed taskruns succeeded",
+		rprt: ResolvedPipelineRunTask{
+			PipelineTask: matrixedPipelineTask,
+			TaskRuns:     []*v1beta1.TaskRun{makeSucceeded(trs[0]), makeSucceeded(trs[1])},
+		},
+		want: false,
+	}, {
+		name: "one matrixed taskrun succeeded",
```

Comment:
how is this different from the test case "one matrixed taskrun running"?

Reply:
good catch, fixing

Reply:
removed the duplicate test case